.github/scripts/tests/tests.sh

```diff
@@ -1,5 +1,220 @@
-#!/usr/bin/env bash
+#!/bin/bash
+# This is script is defined as the following:
+# 1 - We declare the required environment variables
+# 2 - Has the functions defined
+# 3 - Setup the environment and run the tests by using the appropriated functions
 
-set -ex
+set -e
```

Comment:
My suggestion: `set -euo pipefail`

From bash man page:
* -e: Exit immediately if a pipeline returns a non-zero status
* -u: Treat unset variables and parameters other than the special parameters ‘@’ or ‘*’, or array variables subscripted with ‘@’ or ‘*’, as an error when performing parameter expansion
* -o pipefail: If set, the return value of a pipeline is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands in the pipeline exit successfully.

Reply:
Done!